src/PIL/PdfParser.py

```diff
@@ -822,13 +822,11 @@ def get_value(cls, data, offset, expect_indirect=None, max_nesting=-1):
             offset = m.end()
             m = cls.re_stream_start.match(data, offset)
             if m:
+                stream_len_str = result.get(b"Length")
                 try:
```

Comment:
```suggestion
                try:
                    stream_len_str = result.get(b"Length")
```
is IMO the easiest to understand.